### PR TITLE
Code Cleanup - Remove Regions from solution 

### PR DESCRIPTION
**Pull request by jonreding2010**
Removed #Regions from all areas Except SpecFlow and Docs
Removed <Examples> from all areas Except SpecFlow and Docs

#61

### CODE REVIEW

File: Framework/SeleniumUnitTests/ElementHandlerUnitTests.cs [hunk 1]
```diff
@@ -99,16 +99,6 @@ public class ElementHandlerUnitTests : BaseSeleniumTest
         /// <summary>
         /// Unit Test for creating a sorted comma delimited string
         /// </summary>
-        #region SortFromWebElements
-        [TestMethod]
-        [TestCategory(TestCategories.Selenium)]
```
Comment (TroyWalshProf) [inline Framework/SeleniumUnitTests/ElementHandlerUnitTests.cs]: Why did you delete this test?

Comment (jonreding2010) [reply Framework/SeleniumUnitTests/ElementHandlerUnitTests.cs]: didn't mean to, slip of the finger
